rocketpy/Flight.py

```diff
@@ -2531,7 +2508,7 @@ def retrieve_temporary_values_arrays(self):
         return temporary_values
 
     @cached_property
-    def calculate_rail_button_forces(self):
+    def __calculate_rail_button_forces(self):
```

Comment:
Any reason to make this private?

Reply:
1. I just did not want to allow user to get the whole tuple of 4 Function objects.
2. The return is a tuple of 4 elements describing the forces in the rocket coordinates system. This needs to be converted to "rail button" reference system. If we don't convert, we can't call it shear or normal force. I don't think it would be good if we set this "intermediate" result as public.